src/Curators/NewsArticleNotFoundException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Curators;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use Exception;
+
+final class NewsArticleNotFoundException extends Exception
```

Comment:
Is there a reason why we still postfix exception classes with `...Exception`?
I think `NewsArticleNotFound()` is clear enough.

Reply:
Good point, I changed it to `NewsArticleNotFound`